### DISCUSSION TITLE
Borrar última línea archivo output.

### DISCUSSION THREAD

**Discussion by bozzimuc** [IIC2133-PUC/2024-2#63] [Tarea 0] [answered]
Buenas tardes, tengo un pequeño problema que al revisar el output de mi código, aparece una última línea vacía después del output esperado, y no sé cómo eliminarla. Muchas gracias

**Comment by tomastrivino** [comment, marked as answer]
El salto de línea está dado por el carácter `\n`. Puedes mostrar foto de esas últimas líneas que mencionas para cachar mejor?

**Reply by bozzimuc** [reply]
![image](https://github.com/user-attachments/assets/18f50b0a-893c-4f4b-8640-0548155653b3)

Después de las líneas que corresponden a la solución, me aparece una línea vacía al final.

**Reply by bozzimuc** [reply]
@tomastrivino todavía no he podido solucionar el problema. Muchas gracias

**Reply by tomastrivino** [reply]
Eso debería estar bien, no es ningún problema. Fíjate que en los output de los tests es igual.
